c2corg_ui/static/js/card.js

```diff
@@ -130,6 +130,34 @@ app.CardController.prototype.translate = function(str) {
 
 
 /**
+ * Show only one of the area types, the first that is available:
+ * 1) range 2) admin limits 3) country
+ * @param {?Array<Object>} areas
+ * @return {string | null}
+ * @export
+ */
+app.CardController.prototype.showArea = function(areas) {
+  if (!app.utils.isTopoguide(window.location.pathname.substring(1)) && areas) {
```

Comment:
ok... maybe i'd better do the issue you linked and then see how it works with this one.